website/pages/docs/feedback/toast.mdx

```diff
@@ -238,13 +244,4 @@ toast({
 
 ## Props
 
-| Name          | Type                                                                    | Default  | Description                                                          |
-| ------------- | ----------------------------------------------------------------------- | -------- | -------------------------------------------------------------------- |
-| `title`       | `React.ReactNode`                                                       |          | The title of the toast.                                              |
-| `isClosable`  | `boolean`                                                               | `false`  | If `true` adds a close button to the toast.                          |
-| `onClose`     | `function`                                                              |          | Callback function to close the toast.                                |
-| `description` | `React.ReactNode`                                                       |          | The description of the toast.                                        |
-| `status`      | `default`, `success`, `error`, `warning`, `info`                        |          | The status of the toast.                                             |
-| `duration`    | `number`                                                                | `5000ms` | Duration before dismiss in milliseconds, or `null` to never dismiss. |
-| `position`    | `top`, `top-left`, `top-right`, `bottom`, `bottom-left`, `bottom-right` | `bottom` | Position the toast will popup out from.                              |
-| `render`      | `(props: {onClose:() => void, id: string}) => React.ReactNode`          |          | Callback function to display a custom toast.                         |
+<PropsTable of="useToast" />
```

Comment:
The prop table for `position` and `variant` is not visible in the docs. Not sure what could be wrong.

Reply:
I can see `variant` in there but `position` is definitely missing... Not sure why either 🤔